### DISCUSSION TITLE
Akvoredo + Grafana

### DISCUSSION THREAD

**Discussion by gerichkoeke** [akvorado/akvorado#1080] [Q&A]
Good afternoon, what would be the procedure to integrate akvoredo with grafana? What is the default clickhouse username and password?

**Comment by Kemipso** [comment]
Hey qhd-redes,


I assume you found an answer already since this was 3 weeks ago, but for posterity:
Akvorado exposes metrics on http://your-akvorado-url/api/v0/inlet/metrics (see https://demo.akvorado.net/docs/configuration#reporting)
You can use Prometheus, victoria metrics or whichever other collector to scrape this endpoint, and then use that collector as a data source in Grafana.
